packages/eslint-plugin-jsx/rules/jsx-curly-brace-presence/jsx-curly-brace-presence.test.ts

```diff
@@ -25,7 +25,7 @@ const parserOptions = {
 
 const ruleTester = new RuleTester({ parserOptions })
 ruleTester.run('jsx-curly-brace-presence', rule, {
-  valid: parsers.all([].concat(
+  valid: parsers.all(([] as any[]).concat(
```

Comment:
Please check other rules for typing this

Reply:
The test has a `feature` field, but the type `ValidTestCase` hasn't included this. I was trying the type of `parsers.all`, but the type is `any` too.